Pipfile

```diff
@@ -11,6 +11,7 @@ gitpython = "*"
 rebasehelper = "*"
 packitos = "*"
 sh = "*"
+timeout-decorator = "*"
```

Comment:
For consistency add installing `python3-timeout-decorator` to the setup instructions with dnf in the README.